services/notification/notifiers/mixins/message/sections.py

```diff
@@ -819,3 +819,131 @@ async def do_write_section(
                     ),
                 )
             )
+
+
+class TeamPlanWriterSection:
```

Comment:
All other sections are in the form `<Section>SectionWriter` (noticed you did the opposite) and inherit from `BaseSectionWriter`.

I think this would be better placed in a separate file if it won't  inherit from `BaseSectionWriter` (although you could do that and keep the interface).